src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -423,6 +423,57 @@
             </div>
           </div>
 
+        <!-- Quotas -->
+        <div>
+          <legend i18n>Quotas</legend>
+
+          <!-- Max Bytes -->
+          <div class="form-group"
+               [ngClass]="{'has-error': form.showError('max_bytes', formDir)}">
+            <label class="control-label col-sm-3"
+                   for="max_bytes">
+              <ng-container i18n>Max bytes</ng-container>
```

Comment:
I would suggest adding the info here and change the text to the following:
```suggestion
              <ng-container i18n>Max bytes</ng-container>
              <cd-helper>
                <span i18n>Leave it blank or specify 0 to disable this quota.</span>
                <br>
                <span i18n>A valid quota should be greater then 0.</span>
              </cd-helper>
```

Reply:
When creating a new pool, it's natural not to set any quota if a user left the fields empty.
When editing a existed pool, if a user left the fields empty, should we keep previous quota values of pool? Or just disable quota (set to 0) if the field are empty?

Reply:
I think if the user left the field empty, we should set the value to the default.
In this case default is disabled, right?

Reply:
It make sense, since the user clear the field. Thanks!